ts/case-report.ts

```diff
@@ -440,6 +485,7 @@ function displayDataReviewSignOff(fragment: DocumentFragment, qcable: Qcable) {
 
 function displaySignOff(
   fragment: DocumentFragment,
+  caseStopped: boolean, // TODO
```

Comment:
todo?

Reply:
Was a reminder for myself that I missed removing. Removed now